pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/TableViews.java

```diff
@@ -141,28 +146,41 @@ public TableView getExternalView(
   public String getSegmentsStatusDetails(
       @ApiParam(value = "Name of the table", required = true) @PathParam("tableName") String tableName,
       @ApiParam(value = "realtime|offline", required = false) @QueryParam("tableType") String tableTypeStr,
+      @ApiParam(value = "Show segments being replaced: true|false", required = false)
+      @QueryParam("showReplacedSegments") @DefaultValue("true") boolean showReplacedSegments,
```

Comment:
Consider making the name consistent as `replaceReplacedSegments`

Reply:
why not `includeReplacedSegments`?

Reply:
Sorry I meant `includeReplacedSegments` and made a typo